react/components/Slider.js

```diff
@@ -525,20 +526,20 @@ class Slider extends PureComponent {
       duration,
       cursor,
     } = this.props
-    const { enableTransition, dragDistance } = this.state
-    if (!this.perPage) {
-      this.perPage = resolveSlidesNumber(this.props.perPage)
-    }
+    const { enableTransition, dragDistance, firstRender } = this.state
 
     const classes = {
       ...Slider.defaultProps.classes,
       ...classesProp,
     }
 
     const arrayChildren = React.Children.toArray(children)
+    const sliderFrameWidth =
```

Comment:
talvez muda de `Frame` pra `Viewport`, se é o que eu to pensando

Reply:
O elemento `sliderFrame` é o que contém os slides (tanto os que estão sendo mostrados quanto os que não estão), faria sentido ser chamado de `sliderViewport`? Talvez `sliderViewport` seria um nome melhor pra o elemento que é chamado de `selector`

Reply:
bom, podemos pensar em outro nome outra hora. é que `frame` costuma ser usado mais pra animação, pode ser misleading. mas isso pode ser feito em outro PR

Reply:
Faz sentido